### PR TITLE
Logging no incidents found

### LINKED ISSUE #81: Log If no incidents found
### Task

- [x] Add logging to national_rail.py to display message if no incidents found
- [x] Don't load to database if no incidents

### User Story:
As a developer I want to be notified if no incidents were found, so I can understand why the database is not being populated.

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
Logs when there are no new incidents.
Closes #81

### CODE REVIEW

File: national_rail/national_rail.py [hunk 1]
```diff
@@ -15,4 +15,7 @@
 
     incidents_data = transform_national.transform_national_rail_data(FILENAME)
 
-    load_national.load_incidents(incidents_data)
+    if not incidents_data:
+        logging.info("No incidents found.")
+    else:
+        load_national.load_incidents(incidents_data)
```
Comment (A-Faris) [inline national_rail/national_rail.py]:
```suggestion
        load_incidents(incidents_data)
```

File: national_rail/national_rail.py [hunk 2]
```diff
@@ -15,4 +15,7 @@
 
     incidents_data = transform_national.transform_national_rail_data(FILENAME)
```
Comment (A-Faris) [inline national_rail/national_rail.py]:
```suggestion
    incidents_data = transform_national_rail_data(FILENAME)
```

File: national_rail/national_rail.py
Comment (A-Faris) [inline national_rail/national_rail.py:1]: use `from` for imports

**Review by A-Faris — DISMISSED** [review]
All good, just better to use from

**Review by fm1psy — DISMISSED** [review]
Looks good 👍

**Review by fm1psy — APPROVED** [review]
Great

**Pull request merged**